### ISSUE TITLE
Jump to heading as well

### ISSUE THREAD

**Issue by skube** [grantwinney/generate-links-for-headers-in-chrome#9]
My suggestion is: 
I'm used to the way GitHub (clickable) headings work—specifically it updates the browser location bar and thus jumps to it. It would be great if this extension also did that (in addition to copying to clipboard?)

Just my 2¢

**Comment by grantwinney** [comment]
I think that'd be easy enough to add in.. just a simple redirect to the link with the anchor appended.

Just curious, but why do you prefer it? I never really liked the page jumping around, or the second step requiring me to manually copy the URL. Seems like adding it to this extension would be a bit redundant. I'm open to hearing a different point of view though.

**Comment by skube** [comment]
1.  Jumping to the heading will immediately show how the link will behave for someone else
2. Often headings can be obscured by floating navs and whatnot, so it's good to see the outcome
3. GitHub is widely used and could be considered canonical or expected behaviour
4. It would only jump if you click. One could, if they wanted to, right-click and copy link without jumping the page
5. Just because it jumps doesn't mean it can't also copy itself—negating the need to manually copy the URL

**Comment by grantwinney** [comment]
Your first two points are good ones, but GitHub is probably only expected behavior for devs, and I don't know who the userbase for this extension might be. I've noticed other sites, like Confluence, implement it the way I did. All that said though, I think it's 6 of one, half-dozen of another, so we'll try it your way. :)

It'll go live whenever Google decides to approve it... they've been really finnicky and slow to approve updates recently, so hopefully it doesn't take too long. :p
